README.md

```diff
@@ -75,9 +75,26 @@ Then configure any rules you'd like to tweak under the `rules` section:
 ## Contributions
 
 We welcome contributions!
-There are a few ideas in the Issues of this repo.
+There are a few ideas in the [Issues] of this repo.
+
+To get started with how to lint JavaScript, play around with the [AST Explorer].
+
 We use a form of [git-flow]; please create any Pull Requests based on the `develop` branch.
-Please add tests.
+
+Please add tests!
+
+
+## Release Process
+
+This codebase attempts to follow [Semantic Versioning].
+
+1. One or more *approved* Pull Requests are merged to the `master` branch.
```

Comment:
More of a FYI but I'm a big fan of always using `1.` for all items in an MD ordered list to keep it simple since it will always render the correct order.

Reply:
Hm yeah... I like having it be readable in plain-text mode as well, although I suppose having all `1`s isn't super unreadable...